nncf/experimental/onnx/graph/model_transformer.py

```diff
@@ -148,93 +149,117 @@ def select_model_inputs_outputs(model, outputs=None):
 
     def _apply_quantizer_insertion_transformations(self):
         # TODO: optimize
+        self.added_target_edges = Counter()
```

Comment:
I think this variable should be initialized at `__init__` and then update if needed.

Reply:
This counter needs only for quantizer insertion. I think that this place is good. Just need to make it private